pkg/gen/typegen.go

```diff
@@ -987,7 +981,20 @@ func createGroups(definitionsJSON map[string]interface{}, opts groupOpts) []Grou
 				continue
 			}
 			results = append(results, aliasString)
+
+			// "apiregistration.k8s.io" was previously called "apiregistration", so create aliases for backward compat
+			if strings.Contains(fqGroupVersion, "apiregistration.k8s.io") {
```

Comment:
Can we use `HasPrefix`/`TrimPrefix` here like the removed code did?

Reply:
Any particular reason why?

Reply:
I tried, but realized that the substitution is actually in the middle of the string since the aliases are of the form: `kubernetes:Group/Version:Kind`. The other part of the code I altered was just operating on the `Group/Version` segment.

Reply:
@pgavlin Is this a blocker, or ok to merge?

Reply:
I'd still prefer that we be more precise here. Maybe split the string on `:` first?

Reply:
@pgavlin Done